rel/files/couchdb.cmd.in

```diff
@@ -25,8 +25,9 @@ set EMU=beam
 set PROGNAME=%~n0
 set PATH=%PATH%;%COUCHDB_BIN_DIR%
 
-set COUCHDB_QUERY_SERVER_JAVASCRIPT="{{prefix}}/bin/couchjs {{prefix}}/share/server/main.js"
-set COUCHDB_QUERY_SERVER_COFFEESCRIPT="{{prefix}}/bin/couchjs {{prefix}}/share/server/main-coffee.js"
+IF NOT DEFINED COUCHDB_QUERY_SERVER_JAVASCRIPT SET COUCHDB_QUERY_SERVER_JAVASCRIPT="{{prefix}}/bin/couchjs {{prefix}}/share/server/main.js"
```

Comment:
This should work but needs empirical validation, I can do this Monday

Reply:
Thanks!

Reply:
You need to remove the quotation marks:

```batch
 IF NOT DEFINED COUCHDB_QUERY_SERVER_JAVASCRIPT SET COUCHDB_QUERY_SERVER_JAVASCRIPT={{prefix}}/bin/couchjs {{prefix}}/share/server/main.js
```